test_server.py

```diff
@@ -42,11 +44,12 @@ def test_start_server(server_process):
 
 
 def test_response_ok():
-    assert server.response_ok() == STATUS200
+    assert b"200 OK" in server.response_ok().split(CRLF)[0]
```

Comment:
200 OK is also in this string:

```
I just signed up for 200 OK Cupid memberships
```

Is that an HTTP response? This can be a tricky thing to test, but currently you are open to false positives.